### PR TITLE
add a function to find the Node that initiate the drag event

### PR DESCRIPTION
**Pull request by lionelB**
add a getDragInitiator to get the cprrect draggable target

### CODE REVIEW

**Comment by lionelB** [comment]
tests are coming

File: src/drag-drop/dragAndDrop.js [hunk 1]
```diff
@@ -62,6 +62,22 @@ angular.module('org.bonitasoft.dragAndDrop',[])
...
 
+    /**
+     * Try to find the node that initiate the dragEvent
+     * @param  {Node} node the event.target node
+     * @return {Node}      the found node or null
+     */
+    this.getDragInitiatorNode = function getDragInitiatorNode(node) {
+      var currentNode = node;
+      while(currentNode.parentNode) {
+        if (currentNode.getAttribute('draggable') === 'true') {
```
Comment (cpuy) [inline src/drag-drop/dragAndDrop.js]: Will it work if we set data-draggable instead of draggable ?

Comment (lionelB) [inline src/drag-drop/dragAndDrop.js]: no and it should not
to make and element draggrable, you need to set a draggable=true attribute
https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Drag_operations#dragstart

Comment (cpuy) [inline src/drag-drop/dragAndDrop.js]: ok

**Pull request merged**